src/loaders/compute_tools/eggnog/eggnog.py

```diff
@@ -44,9 +43,13 @@ def _run_eggnog_single(
 
     # Save run info to a metadata file in the output directory for parsing later
     metadata = {'source_file': str(source_file),
-                'input_type': INPUT_TYPE}
-    with open(metadata_file, 'w') as f:
-        json.dump(metadata, f, indent=4)
+                'input_type': INPUT_TYPE,
+                'data_id': data_id,
+                'tool_name': 'eggnog',
+                'version': '2.1.12',
+                'command': command,
+                'reference_db': '5.0.2',}
```

Comment:
I would use the same reference db structure as I suggested earlier

Reply:
👍

Reply:
:+1: